src/site/content/en/_redirects.yaml

```diff
@@ -173,3 +173,8 @@ redirects:
 
 - from: /discover-installable/
   to: /progressive-web-apps/#installable
+
+# Redirect blog content
+
+- from: /sms-receiver-api-announcement
+  to: /webotp
```

Comment:
[This comment](https://github.com/GoogleChrome/web.dev/pull/2303#issuecomment-597145853) from @beaufortfrancois says that the API name is `Web OTP` rather than `WebOTP` which suggests that the URL should be `/web-otp` not `/webotp`

Reply:
+1 to /web-otp/

Reply:
I've sent an email to the spec author. I could have sworn I saw a draft spec with 'WebOTP'.